src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -926,13 +926,14 @@ TEST(Aktualizr, APICheck) {
     Aktualizr aktualizr(conf, storage, up, sig);
     boost::signals2::connection conn = aktualizr.SetSignalHandler(f_cb);
     aktualizr.Initialize();
-    for (int i = 0; i < 100; ++i) {
+    for (int i = 0; i < 101; ++i) {
```

Comment:
Why?

Reply:
To make 100 cycles, otherwise we make only 99

Reply:
Um... no.

Reply:
I can change it to 'i <=100 '

Reply:
There are 100 numbers in the range [0, 99]. `i < 100` is exactly what we want.

Reply:
@patriotyk happens to me as well sometimes :)